video/out/opengl/video.c

```diff
@@ -3006,13 +3006,13 @@ void gl_video_render_frame(struct gl_video *p, struct vo_frame *frame,
     if (!has_frame || !mp_rect_equals(&p->dst_rect, &target_rc)) {
         struct m_color c = p->clear_color;
         float color[4] = {c.r / 255.0, c.g / 255.0, c.b / 255.0, c.a / 255.0};
-        p->ra->fns->clear(p->ra, target.tex, color, &target_rc);
+        p->ra->fns->clear(p->ra, target.tex, color);
```

Comment:
Well, I'm trying hard to avoid `if (vulkan) { do one thing } else { do other thing }` wherever possible, so ideally I think we'd make the APIs similar if we can. Besides, I don't want to keep track of where and when I need to clear based on some weird backend-specific decision logic like “is explicit clear supported? is per-renderpass clear supported?”. I'd rather introduce one mechanism that works for both if we can, and I think if we just move clearing to the renderpass it would work for both?